spec/fixtures/models/image_dir.rb

```diff
@@ -0,0 +1,5 @@
+class ImageDir
+  include DirModel
+
+  file :my_image, path: "{{root}}/{{dir}}/{{sub_dir}}", name: :my_image_name
```

Comment:
:warning:  btw, what are you thoughts on this syntax?

```ruby
file :my_image, path: -> { "#{{root}}/#{{dir}}/#{{sub_dir}}" }, name: -> { "my_#{image}_name" }
```
pro: no new template language, uses natural ruby instead.
con: interpolating the methods available might be complicated (but possible, with some magic)